### ISSUE TITLE
Hide Resize Grip without disabling Resizing

### ISSUE THREAD

**Issue by zawata** [ocornut/imgui#2254]
I want to remove the resize grip without actually disabling resizing. It's slightly interfering with my window layout.

Here's what I've tried/looked for thus far.
The `ImGuiWindowFlags_NoResize` flag obviously disables resizing the windows.
I don't exactly know how `ImGuiStyleVar_GrabMinSize` and `ImGuiStyleVar_GrabRounding` work or what they do but setting either to 0 via `PushStyleVar` Doesn't do anything

I can achieve this by the following code: 
```
PushStyleColor(ImGuiCol_ResizeGrip, GetColorU32(ImGuiCol_WindowBg));
PushStyleColor(ImGuiCol_ResizeGripActive, GetColorU32(ImGuiCol_WindowBg));
PushStyleColor(ImGuiCol_ResizeGripHovered, GetColorU32(ImGuiCol_WindowBg));
``` 
but this feels like a round-about method as it still actually draws the grips they just aren't colored and it still changes the mouse-cursor. 

If there is not an official way to do this(which, according to the draw code in `begin()`, it seems like there isn't ), is there a way to disable changing the mouse cursor on the bottom left?

**Comment by ocornut** [comment]
> I don't exactly know how ImGuiStyleVar_GrabMinSize and ImGuiStyleVar_GrabRounding work or what they do but setting either to 0 via PushStyleVar Doesn't do anything

They are the grab used by e.g. sliders. Comment says "Minimum width/height of a grab box for slider/scrollbar.". If unsure, instead of using `PushStyleVar()` use the style editor to dynamically adjust the value and see the effect it has on your global style.

To make it invisible for a certain window just use:
```
PushStyleColor(ImGuiCol_ResizeGrip, 0);
```
etc.

There's no way to change the mouse cursor but you can perfectly ignore the cursor value or overwrite it before calling EndFrame():
```
if (ImGui::GetMouseCursor() == ImGuiMouseCursor_ResizeNWSE)
  ImGui::SetMouseCursor(ImGuiMouseCursor_Arrow);
```

**Comment by zawata** [comment]
Ah, I was only showing the window I was working on and it didn't have the scroll bars turned on. also must have missed that comment...

I forgot we worked in RGBA not RGB so i assumed 0 would have made it black. 

Helpful with the mouse stuff, I just learned my desktop manager uses fists instead of diagonal resize-arrows(though it uses horizontal and vertical resize arrows correctly...) which made me think it was a different kind of cursor. 

Thanks for the help and fast response. Happy Holidays!
